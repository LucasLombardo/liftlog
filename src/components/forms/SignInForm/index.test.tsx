import { SignInForm } from "./index";
import { render, screen } from "@testing-library/react-native";

describe("SignInForm", () => {
  it("component matches snapshot", () => {
    render(<SignInForm onSubmit={() => null} disabled={false} />);
    expect(screen).toMatchSnapshot();
  });
});
