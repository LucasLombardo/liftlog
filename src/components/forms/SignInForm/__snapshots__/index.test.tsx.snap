// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInForm component matches snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "display": "flex",
    }
  }
>
  <View
    style={
      [
        {
          "borderColor": "black",
          "borderWidth": 1,
          "marginTop": 16,
          "maxWidth": 420,
          "minWidth": 240,
          "paddingHorizontal": 20,
          "paddingVertical": 10,
          "width": "70%",
        },
        null,
      ]
    }
  >
    <Text
      style={
        {
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 12,
        }
      }
    >
      Sign in
    </Text>
    <View>
      <Text
        style={
          {
            "fontSize": 16,
          }
        }
      >
        Email
      </Text>
      <TextInput
        accessibilityLabel="Email"
        editable={true}
        onBlur={[Function]}
        onChangeText={[Function]}
        style={
          {
            "borderColor": "black",
            "borderWidth": 1,
            "marginBottom": 10,
            "padding": 4,
          }
        }
        value=""
      />
    </View>
    <View>
      <Text
        style={
          {
            "fontSize": 16,
          }
        }
      >
        Password
      </Text>
      <TextInput
        accessibilityLabel="Password"
        editable={true}
        onBlur={[Function]}
        onChangeText={[Function]}
        secureTextEntry={true}
        style={
          {
            "borderColor": "black",
            "borderWidth": 1,
            "marginBottom": 10,
            "padding": 4,
          }
        }
        value=""
      />
    </View>
    <View
      accessibilityRole="button"
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          [
            {},
          ]
        }
      >
        <Text
          style={
            [
              {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
            ]
          }
        >
          Submit
        </Text>
      </View>
    </View>
  </View>
</View>
`;
